### PR TITLE
add 3 traits to scraped data files

### PR DESCRIPTION
**Pull request by ehwenk**
add `plant_photosynthetic_organ`, `leaf_lamina_posture` and `leaf_margin_posture` to scraped flora data files

### CODE REVIEW

File: data/ABRS_2023/metadata.yml [hunk 1]
```diff
@@ -349,6 +352,36 @@ traits:
   basis_of_value: measurement
   replicates: .na
   methods: Maximum plant vegetative height of a mature individual.
+- var_in: plant_photosynthetic_organ
+  unit_in: mm
+  trait_name: plant_photosynthetic_organ
+  entity_type: species
+  value_type: value_type
+  basis_of_value: measurement
```
Comment (yangsophieee) [inline data/ABRS_2023/metadata.yml:360]: Should these be `expert_score` as they are categorical values (relevant to the rest of the studies)?

File: data/ABRS_2023/metadata.yml [hunk 2]
```diff
@@ -349,6 +352,36 @@ traits:
   basis_of_value: measurement
   replicates: .na
   methods: Maximum plant vegetative height of a mature individual.
+- var_in: plant_photosynthetic_organ
+  unit_in: mm
```
Comment (yangsophieee) [inline data/ABRS_2023/metadata.yml:356]: Should units be `.na`?

**Review by yangsophieee — APPROVED** [review]
Awesome work!

File: data/ABRS_2023/metadata.yml (hunk 1 above)
Comment (ehwenk) [reply data/ABRS_2023/metadata.yml:360]: yes, I didn't copy and paste correctly

File: data/ABRS_2023/metadata.yml (hunk 2 above)
Comment (ehwenk) [reply data/ABRS_2023/metadata.yml:356]: yes, fixed

**Pull request merged**